### DISCUSSION TITLE
Chapter display in Android Auto

### DISCUSSION THREAD

**Discussion by Travillion** [advplyr/audiobookshelf#1358] [Ideas]
First off thanks for the amazing service!

I would love if we could get the currently playing chapter to display in Android Auto. I know some people, myself included, have gone to great lengths to get proper chapter tags, and it would be awesome for those to be displayed. There is a lot of space beneath the book title where it could easily fit.

**Comment by tomazed** [comment]
Hello Travillion,

Maybe this idea would find better success in the mobile dedicated git
https://github.com/advplyr/audiobookshelf-app

**Comment by Travillion** [comment]
Thanks for the tip! I looked in a couple of different places trying to find
the right spot. As is obvious, I'm not a regular on GitHub....
